packages/contracts/deploy-config/local.ts

```diff
@@ -1,15 +1,15 @@
 const config = {
-  l1BlockTimeSeconds: 15,
-  l2BlockGasLimit: 15_000_000,
-  l2ChainId: 17,
-  updateGaslimitBlock: 10,
-  eigenDaBlock: 100,
+  numDeployConfirmations: 1,
+  gasPrice: 150_000_000_000,
+  l1BlockTimeSeconds: 12,
+  l2BlockGasLimit: 30_000_000,
+  l2ChainId: 5000,
```

Comment:
`L2ChainID` of Local network looks wrong.

Reply:
done